packages/ripple-address-codec/.eslintrc.js

```diff
@@ -39,6 +39,7 @@ module.exports = {
     'jsdoc/require-throws': 'off',
     'jsdoc/require-jsdoc': 'off',
     'jsdoc/require-param': 'off',
+    'jsdoc/check-examples': 'off',
```

Comment:
Why is this turned off now?

Reply:
This rule isn't supported in the new version of `eslint` - ripple-address-codec:   145:1  warning  This rule cannot yet be supported for ESLint 8; you should either downgrade to ESLint 7 or disable this rule. The possibility for ESLint 8 support is being tracked at https://github.com/eslint/eslint/issues/14745  jsdoc/check-examples